### PR TITLE
Release Operator v0.0.15

### PR DESCRIPTION
**Pull request by jomkz**
This release includes the following enhancements, bug fixes and documentation updates.

Enhancements

* #184 - Add Property to Disable Admin User
* #194 - Add support for proxies in the repo-server deployment

Bug Fixes

* #186 - Pods stuck in crashloop after update to 0.0.14
* #187 - Fix issue with replacing the resource configuration
* #191 - Missing gpg configmap after update argocd-operator 0.0.13 to 0.0.14

Documentation

* #189 - Add documentation for the resourceInclusions property
* #190 - Fixed port-forward command
* #195 - Fixed wrong `TLS` key example for server route

### CODE REVIEW

**Review by bigkevmcd — APPROVED** [review]
Looks ok to me, it would perhaps be useful to have a script to do this?

**Comment by jmckind** [comment]
Yes, that is on my list of things that need automation!

File: deploy/catalog_source.yaml [hunk 1]
```diff
@@ -4,6 +4,6 @@ metadata:
   name: argocd-catalog
 spec:
   sourceType: grpc
-  image: quay.io/jmckind/argocd-operator-registry@sha256:5ee18ccd4191bfaf0891e89de026a0dbcae2c8d4a150bea2c6488f8bd2e69943
+  image: quay.io/jmckind/argocd-operator-registry@sha256:69148d9a3fd9d6d3b47fa9a45de3bd8bd7ecc932cb8cbb76ba4aea5f603d0567
```
Comment (bigkevmcd) [inline deploy/catalog_source.yaml:7]: I merged a change that dropped logging that should not be in the released image.

**Comment by daufinsyd** [comment]
Hi :-)
Do you have any ETA when the new version would be released ?
Thanks

**Comment by djkormo** [comment]
Will operator 0.15  support argocd 1.8 ?

**Comment by EagleIJoe** [comment]
> Will operator 0.15  support argocd 1.8 ?

Unlikely as the code change for the upgrade is this commit: https://github.com/argoproj-labs/argocd-operator/commit/ab24cb9e8229cd73b6b5dcd44317bfdf082abc91

**Comment by jomkz** [comment]
Closing this, will create a new PR for the upcoming release.

**Pull request closed without merging**